buildSrc/src/test/java/com/osacky/flank/gradle/integration/TestFixtures.kt

```diff
@@ -1,8 +1,27 @@
 package com.osacky.flank.gradle.integration
 
+import org.gradle.testkit.runner.GradleRunner
 import org.junit.rules.TemporaryFolder
 import java.io.File
 
 fun TemporaryFolder.setupFixture(fixtureName: String) {
   File(this::class.java.classLoader.getResource(fixtureName)!!.file).copyRecursively(newFile(fixtureName), true)
 }
+
+internal fun makeGradleFile(where: TemporaryFolder, stringProvider: () -> String) = where
+  .newFile("build.gradle")
+  .writeText(stringProvider().trimMargin())
+
+internal fun gradleRun(block: GradleRunFladle.() -> Unit) = GradleRunFladle().apply(block).run {
```

Comment:
Can this be made in to a simple function?
Or maybe I'm missing something. What is the benefit of this syntax?
```suggestion
internal fun gradleRun(arguments: List<String> = emptyList(), projectDir: File? = null))
```

Reply:
Sure, no problem, will change that 👍